stats.cpp

```diff
@@ -1,5 +1,31 @@
 #include "stats.h"
 
-Stats Statistics::ComputeStatistics(const std::vector<___>& ) {
+Stats Statistics::ComputeStatistics(const std::vector<double>& values) {
     //Implement statistics here
+    Stats stats;
```

Comment:
This is in a different namespace. Use the namespace defined in the header file to make this work.